Procfile

```diff
@@ -0,0 +1 @@
+worker: node dist/index.js
```

Comment:
Why did you go for worker instead of a normal web process? At least for now it seems that a web process would be enough, considering we don't take advantage of the possibilities of the worker.

Reply:
I just assumed that a web worker was for HTTP hosted things and that the worker was for everything else.
It's also how I have my other bot I built deployed, with a website running on the web process and the bot running on the worker process.

Reply:
Makes sense. I just wanted to make sure I wasn't missing something. The only difference from the code side is that you wouldn't need a Procfile with a web resource, since you could add the command to start. Really minor though, so I think it's fine to stay with the worker.